src/kontrol/prove.py

```diff
@@ -494,6 +513,139 @@ def _init_cterm(
         return init_cterm
 
 
+def _init_cse_cterm(
+    empty_config: KInner,
+    contract_name: str,
+    program: KInner,
+    method: Contract.Method | Contract.Constructor,
+    fields: FrozenDict,
+    *,
+    setup_cterm: CTerm | None = None,
+    calldata: KInner | None = None,
+    callvalue: KInner | None = None,
+) -> CTerm:
+    contract = KEVM.account_cell(
+        KVariable('CONTRACT'),
+        KVariable('CONTRACT_BAL'),
+        program,
+        KVariable('CONTRACT_STORAGE'),
+        KVariable('CONTRACT_STORAGE'),
```

Comment:
It looks like we're duplicating a bunch of code, I wonder if we could reuse the `init_cterm` and just pass in the flag instead? Also, can we just make it so that whenever you call a non `test_*` function, this is the default?There is no other use-case of `init_cterm` on a non-`test_*` function _except_ for CSE, so we should just make that function do this whne this flag is passe.d